back-end/Models/comments/comment.js

```diff
@@ -0,0 +1,41 @@
+const connection = require('../../db');
+require('dotenv').config();
+
+//Add new comment
+const addComment=(req,res)=>{
+  let sql = "INSERT INTO comment SET ?";
+  let comment = req.body.comment;
+  connection.query(sql, comment,(err, results) => {

```

Comment:
you should use `const` instead of `let`, also you could change the variable names into `query` and `data`.

The SQL query is supposed to do what exactly?  why aren't you using something similar to this to insert `INSERT INTO Customers (CustomerName,  Country)
VALUES ('Cardinal', 'Tom B. Erichsen', 'Norway');`

You are supposed to add more information about the comment, like the `post_id` and `user_id`